Adding perl5

jonasbn opened the pull request:
- [X] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md

I am:
  - [ ] Suggesting edits to an existing Topic page
  - [X] Curating a new Topic page

***********EDITING AN EXISTING TOPIC PAGE************

I'm suggesting these edits to an existing topic:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [ ] Content (and my changes are in `index.md`)

Please explain why these changes are necessary:


************CURATING A NEW TOPIC PAGE************

- [X] I've formatted my changes as a new folder directory, named for the topic as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]`)
- [X] My folder contains a `*.png` image (if applicable) and `index.md`
- [X] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Please explain why you think this Topic page should be curated:

I miss Perl being on the list. There are plenty of Perl projects on Github and it is becoming the de facto SCM for CPAN distributions and Perl projects in general.

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

nayafia commented on topics/perl/index.md:
```diff
@@ -0,0 +1,12 @@
+url: perl.org
+wikipedia_url: https://en.wikipedia.org/wiki/Perl
+---
+Perl 5 is a highly capable and feature-rich programming language. Perl 5 runs on over 100 platforms ranging from portables to mainframes and everything in between. The can be used for almost al facets of programming from rapid prototyping and large scale development projects.
```
A few typos and stylistic edits, I'd suggest:

> Perl 5 runs on over 100 platforms ranging from portables to mainframes. The can be used for almost all facets of programming, from rapid prototyping and large-scale development projects.

nayafia replied:
@jonasbn could you make this edit before I merge? ☝️

jonasbn replied:
Done

nayafia commented:
@jonasbn thanks for adding this! Would you like to add an image, as well?

jonasbn commented:
Hi @nayafia,

The problem with a image is that Perl has several images/logos associated. I would love to use the one displayed on [this page](https://github.com/perl). The raptor displayed there is based on [this image](https://github.com/pfig/perl-raptor). The attribution requirements does however not really seem possible in this situation, do you have any experience with this?

Alternatively there is the camel displayed [on wikipedia](https://en.wikipedia.org/wiki/Perl), this is however [more confusing](http://www.onlamp.com/pub/a/oreilly/perl/usage/ ) when it comes to use.

I am investigating with the Perl community how to go about this, but for now I do not want to make use of an image without proper permission.

jonasbn commented:
Hi @nayafia 

After some investigation, SMTP and IRC I have come to the conclusion that the community-create Perl 5 logo already used on GitHub is the best suited logo for this case - I hope you concur.

nayafia commented:
@jonasbn yep, sounds good! (looks a wee bit fuzzy, if you have a higher-res image, but otherwise this is fine)

I just had another few edits in-line, if you could correct those we can merge 👍

nayafia commented:
@jonasbn ah there are also a few broken tests re: naming conventions (redundant alias and image file name), if you could fix those.

jonasbn commented:
Hi @nayafia 

I am on it, thanks for you patience

jonasbn

jonasbn commented:
Hi @nayafia 

I have:
- [X] applied you proposed text changes
- [X] updated the images
- [X] addressed the tests 

Let me know if there is anything else

MikeMcQuaid commented:
Thanks for your first contribution to github/explore, @jonasbn! Without people like you submitting PRs we couldn't run this project. You rock!
